docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -6,12 +6,27 @@ description: Release Notes for 100ms JavaScript SDK
 
 ## Latest Versions
 
-| Package                    | Version                                                                                                                                |
-| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| @100mslive/hms-video-store | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-store.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-store) |
-| @100mslive/react-sdk       | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
+| Package                                | Version                                                                                                                                |
+| -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| @100mslive/hms-video-store             | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-store.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-store) |
+| @100mslive/react-sdk                   | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2022-05-31
+
+Released: `@100mslive/hms-video-store@0.3.0`, `@100mslive/react-sdk@0.1.0`, `@100mslive/hms-video-react@0.4.0`
+
+Fixed:
+
+-   Video freezing when same video track is played in multiple video elements and one of them is removed
+-   Incorrect device showing in device selection when initially joined as muted.
+
+Breaking:
+
+-   All the requests(RoleChange, track mute/unmute etc) will have peer/requestBy as optional now.
```

Comment:
how do we make this more generic? will it be understandable?

Reply:
let's add examples?

For Example:
- In the chat message object, message.sender is now optional
- in unmute request, request.requestedBy is optional